docs/zookeeperAdmin.html

```diff
@@ -1446,7 +1446,17 @@ <h3 class="h4">Configuration Parameters</h3>
               </pre>
 </dd>
 
-        
+<dt>
+<term>tcpKeepAlive</term>
+</dt>
+<dd>
+<p>(Java system property: <strong>zookeeper.tcpKeepAlive</strong>)</p>
```

Comment:
it would be great to provide a brief explanation for users why they may want to do this. without them needing to refer to the JIRA.